machineconfiguration/v1/register.go

```diff
@@ -0,0 +1,47 @@
+package v1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	GroupName     = "machineconfiguration.openshift.io"
+	GroupVersion  = schema.GroupVersion{Group: GroupName, Version: "v1"}
+	schemeBuilder = runtime.NewSchemeBuilder(addKnownTypes, corev1.AddToScheme)
```

Comment:
why corev1?

Reply:
I guess this part was just copied from somewhere else when the MC api was created (see e.g. https://github.com/openshift/api/blob/master/quota/v1/register.go#L13, which is similar to this one here in that it doesn't register a corev1 type directly, but the types in types.go import and depend on it.)

We depend on corev1 in our types.go as well - can I remove the import here nonetheless?

If so, this will be done as follow up. I'll add some `FIXME` comments in this PR.

Reply:
ack